moped-editor/src/utils/activityLogFormatters/mopedPhasesActivity.js

```diff
@@ -0,0 +1,74 @@
+import EventNoteIcon from "@material-ui/icons/EventNote";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatPhasesActivity = (change, phaseList, subphaseList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_proj_phases"];
+
+  const changeIcon = <EventNoteIcon />;
+  const phase = phaseList[change.record_data.event.data.new.phase_id];
+  const subphase = subphaseList[change.record_data.event.data.new.subphase_id];
+  const phaseText = {
+    text: phase,
+    style: "boldText",
+  };
+  const subphaseText = {
+    text: ` - ${subphase}`,
+    style: "boldText",
+  };
+
+  // add a new phase
+  if (change.description.length === 0) {
+    return {
+      changeIcon,
+      changeText: [
+        { text: "Added ", style: null },
+        phaseText,
+        // include subphase name if one exists
+        ...(subphase ? [subphaseText] : []),
+        { text: " as a new phase.", style: null },
+      ],
+    };
+  }
+
+  // delete an existing phase
+  if (change.description[0].field === "is_deleted") {
+    return {
+      changeIcon,
+      changeText: [
+        { text: "Deleted the phase ", style: null },
+        phaseText,
+        // include subphase name if one exists
+        ...(subphase ? [subphaseText] : []),
+      ],
+    };
+  }
+
+  // Multiple fields in the moped_proj_funding table can be updated at once
```

Comment:
moped_proj_funding --> moped_proj_phases

Reply:
going to merge this but i will fix this typo in another pr!